cli/src/pcluster/config/cluster_config.py

```diff
@@ -1629,16 +1629,140 @@ def __init__(
         **kwargs,
     ):
         super().__init__(**kwargs)
-        self.instance_type = Resource.init_param(instance_type)
         self.max_count = Resource.init_param(max_count, default=DEFAULT_MAX_COUNT)
         self.min_count = Resource.init_param(min_count, default=DEFAULT_MIN_COUNT)
         self.spot_price = Resource.init_param(spot_price)
         self.disable_simultaneous_multithreading = Resource.init_param(
             disable_simultaneous_multithreading, default=False
         )
-        self.__instance_type_info = None
         self.efa = efa or Efa(enabled=False, implied=True)
         self.schedulable_memory = Resource.init_param(schedulable_memory)
+        self.__instance_types_with_instance_storage = []
+        self.__instance_type_info_map = {}
```

Comment:
why are you using two underscore? I'd stay consistent with the rest of the code using a single underscore for the private attributes.